website/screens/principles/component-lifecycle/ComponentLifecycle.tsx

```diff
@@ -0,0 +1,234 @@
+    title: "Introduction",
+    content: (
+      <>
+        <DxcParagraph>
+          The lifecycle of a component within a Design System comprises several
+          stages, spanning from its creation to eventual retirement. Starting
+          with the design and development phases, components are conceptualized,
+          designed and implemented. Once in use, they undergo real-world testing
+          and user feedback to identify usability issues and potential
+          improvements. As the system progresses, components may be upgraded or
+          even withdrawn to meet ever-evolving standards and user requirements.
+        </DxcParagraph>
+        <DxcParagraph>
+          Each of the stages can be understood as milestones that summarize the
+          maturity lifecycle of the components in Halstack and have their
+          implications or requirements which need to be fulfilled to move from
+          one to another. This process is non-linear, which means that a
+          component can go backwards if the situation requires so.
+        </DxcParagraph>
+        <DxcParagraph>
+          Adhering to this lifecycle ensures that our designers and developers
+          uphold a well-organized Design System, facilitating a smooth and
+          intuitive user experience.
+        </DxcParagraph>
+      </>
+    ),
+  },
+  {
+    title: "Experimental",
+    content: (
+      <>
+        <DxcParagraph>
+          The component is ready for preliminary usage. It meets most maturity
+          criteria, and use is encouraged for most scenarios. It may undergo
+          last minute changes from its first uses, unexpected scenarios and
+          actual user experience, which can sometimes be hard to estimate.
+        </DxcParagraph>
+      </>
+    ),
+    subSections: [
+      {
+        title: "Implications",
+        content: (
+          <DxcBulletedList>
+            <DxcBulletedList.Item>
+              <strong>Partial documentation.</strong> The documentation of the
+              component may be incomplete, with missing parts, examples or even
+              entire pages.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              <strong>Partial implementation.</strong> The component may lack
+              some features or new ones may be incoming. Regardless of this, all
+              the released functionality is fully tested, including visual
+              regression testing, snapshot testing, unit test, etc.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              Already released and available in the next version of the Halstack
+              npm library.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              Already released and available in our design tools.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              The component API is subject to change.
+            </DxcBulletedList.Item>
```

Comment:
How do you see to change `The component API is subject to change` to `The component API could be subject to change` It's a silly change but the other way seems that for sure, it will be subject to changes.